src/web/modules/gateway_accounts/detail.njk

```diff
@@ -111,6 +111,13 @@
           <dd class="govuk-summary-list__actions"></dd>
         </div>
     {% endif %}
+    {% if account.allow_moto != undefined %}
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">MOTO Payments enabled</dt>
+          <dd class="govuk-summary-list__value">{{ account.allow_moto | string | capitalize }}</dd>
+          <dd class="govuk-summary-list__actions"></dd>
+        </div>
+    {% endif %}
```

Comment:
Possibly don't need this section in the summary table because moto payments has its own section on this page. I think the ultimate goal is to split out more of these settings into sections so there isn't just this massive list. Not a biggie though.